<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <node name="roboclaw1" pkg="roboclaw_driver" type="roboclaw_node.py">
        <param name="test_mode" type="bool" value="true"/>
        <param name="dev_name" type="str" value="/dev/ttyACM0"/>
        <param name="baud" type="int" value="115200"/>
        <param name="address" type="int" value="128"/>
        <param name="loop_hz" type="int" value="10"/>
        <param name="deadman_secs" type="int" value="10"/>
        <param name="speed_cmd_topic" type="str" value="base_node/speed_command"/>
    </node>
</launch>